LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs

```diff
@@ -32,4 +32,6 @@
 };
 
 var configuration = NetworkServerConfiguration.CreateFromEnvironmentVariables();
+// when running in local debug mode we explicitely disable IoT Edge
+configuration.RunningAsIoTEdgeModule = false;
```

Comment:
I don't get the fix. This is not only applied for local development, but also to any docker image being built no?

I would rather leave the code as is, but add something in the docs for the local development

Reply:
Woops. you are totally right, I was remembering this project was only for local dev and I was obviously wrong. Thanks for the catch.

What do you think to execute this line if we have a debugger attached? This typically should not occur in normal scenarios?

Reply:
I would personally prefer to have cleaner code, therefore I would go for completely removing it and just adding a documentation note. As an alternative we might make use of launchsettings.json and appsettings.development.json for any dev related parameter like this

Reply:
I added a new parameters 'LOCAL_DEVELOPMENT' that will disable edge when set to true, If agreed I will also edit the [docs](https://azure.github.io/iotedge-lorawan-starterkit/2.1.0/user-guide/devguide/#debugging-in-visual-studio-outside-of-iot-edge-and-docker)

Reply:
Replied in another comment. Let's move the discussion there. It's fine to have local_development but we can simplify code